### ISSUE TITLE
Detect UTF8-BOM files during validate

### ISSUE THREAD

**Issue by mosbth** [mosbth/dbwebb-cli#4]
Saving files as UTF8-BOM (or UTF-16 BOM) gives problems. Make a test that detects these files during validation process and provide a warning.

**Comment by mosbth** [comment]
Seems like a nice check to avoid problems.

**Comment by mosbth** [comment]
```
18:12 <@foiki> vi borde lägga till så att dbwebb-cli kollar om de är UTF-8 || ascii text || ISO-8859 text
               oxå. och failar studenten om den inte har skrivit i ascii text eller utf-8                
```

**Comment by mosbth** [comment]
The command `file -i` could be a solution.

**Comment by mosbth** [comment]
Solved v1.9.25

**Comment by foikila** [comment]
Like.
